lndr-backend/src/Lndr/EthereumInterface.hs

```diff
@@ -163,8 +163,8 @@ interpretUcacLog change = do
 -- | Verify that a settlement payment was made using a 'txHash' corresponding to
 -- an Ethereum transaction on the blockchain and the associated addresses and
 -- eth settlement amount.
-verifySettlementPayment :: SettlementVerification -> LndrHandler ()
-verifySettlementPayment (SettlementVerification creditorAddr debtorAddr (Just txnHash) settlementValue) = do
+verifySettlementPayment :: Maybe TransactionHash -> Address -> Address -> Integer -> LndrHandler ()
```

Comment:
This should not be a `Maybe TransactionHash`. There is nothing to verify if there is no txHash.

Reply:
BilateralCreditRecord has txHash :: Maybe TransactionHash. What is the best way to get this from the BCR?